apicast/src/policy/load_configuration.lua

```diff
@@ -0,0 +1,33 @@
+local _M = require('policy').new('Load Configuration')
+
+local configuration_loader = require('configuration_loader').new()
+local configuration_store = require('configuration_store')
+
+local new = _M.new
+
+function _M.new(...)
+    local policy = new(...)
+    policy.configuration = configuration_store.new()
```

Comment:
As I commented above for a different module, I think it'd be good to inject the configuration.

Reply:
Agree. We have no facility how to inject stuff to the `new` method. But definitely would be useful to pass something around.